Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 import org.yaml.snakeyaml.Yaml;
 
-def pipeutils, streams, official, official_jenkins, developer_prefix
+def pod, pipeutils, streams, official, official_jenkins, developer_prefix
```

Comment:
It doesn't fix anything actually, but I like to use `def` consistently when declaring variables because it's more akin scope wise to what we're used to in other languages.

Reply:
I dropped that commit for now but will circle back after migration.